setupext.py

```diff
@@ -452,24 +452,6 @@ def get_package_data(self):
         }
 
 
-class Tests(OptionalPackage):
```

Comment:
We probably need to put this back to package the handful of images that we still use in the tests or are they now packaged by default?

Reply:
As far as I remember those images were not image comparison tests for `baseline images`. So, we kept in the `test_data` folder. Then tests were passing. So, do we need this to revert it?

Reply:
These control what goes into the build artifacts (the sdist and the wheels), if you do `pip install -ve .` you are running against the tree and things might work even if the packaging meta data is not correct.